src/Symfony/Component/Security/Core/Authorization/DebugAccessDecisionManager.php

```diff
@@ -1,98 +1,18 @@
 <?php
 
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
 
 namespace Symfony\Component\Security\Core\Authorization;
 
-use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
-
 /**
- * Decorates the original AccessDecisionManager class to log information
- * about the security voters and the decisions made by them.
- *
  * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ * @author Alessandro Lai <alessandro.lai85@gmail.com>
  *
  * @internal
+ * @deprecated Since 3.3. This is a placeholder for the old class, that got renamed into TraceableAccessDecisionManager; 
+ *  this is not a BC break since the class is internal, this placeholder is here just to help backward compatibility 
+ *  with older SecurityBundle versions. 
  */
-class DebugAccessDecisionManager implements AccessDecisionManagerInterface
+class DebugAccessDecisionManager extends TraceableAccessDecisionManager
```

Comment:
Test failing locally... Maybe I'm missing something, but how is it supposed to work like that? It's defining the alias before declaring the class, it says that the Traceable class is missing!